### ISSUE TITLE
gintro install fails with latest gstreamer from gitlab

### ISSUE THREAD

**Issue by StefanSalewski** [StefanSalewski/gintro#138]
```
type
  MiniObject00* {.pure.} = object
    `type`*: GType
    refcount*: int32
    lockstate*: int32
    flags*: uint32
    copy*: MiniObjectCopyFunction
    dispose*: MiniObjectDisposeFunction
    free*: MiniObjectFreeFunction
    privUint*: uint32
    privPointer*: pointer
```

The copy field was a plain pointer before. Now we have three data types in the GstMiniObject, which seems to be one more as the bindings generator currently can handle.

**Comment by StefanSalewski** [comment]
Try nimble install gintro@#head

Plain fix was

```
    block myb:
      for a in 1 .. 3: # process clustered symbols into one single type section
        if delayedSyms.len > a:
```

increasing the maximum cluster size by one, which is needed for gst MiniObject fields.

**Comment by StefanSalewski** [comment]
Should be fixed in v0.9.4 now.
Will close this issue, please reopen if necessary.
